cradle-core/src/main/java/com/exactpro/cradle/CradleStorage.java

```diff
@@ -318,6 +319,22 @@ public BookInfo refreshPages(BookId bookId) throws CradleStorageException, IOExc
 		return book;
 	}
 
+	/**
+	 * @param bookId book of removed pages
+	 * @return collection of removed pages for given book
+	 * @throws CradleStorageException
+	 */
+	public Collection<PageInfo> listRemovedPages(BookId bookId) throws CradleStorageException {
```

Comment:
name listRemovedPages assumes that it only loads removed pages.
rename it to getAllPages, rename proxy  methods as well